mmdet/models/detectors/two_stage.py

```diff
@@ -190,6 +190,10 @@ def simple_test(self, img, img_metas, proposals=None, rescale=False):
 
         x = self.extract_feat(img)
 
+        # get origin input shape to onnx dynamic input shape
+        if torch.onnx.is_in_onnx_export():
+            img_metas[0]['img_shape_for_onnx'] = img.shape[2:]
```

Comment:
Argument `img_shape` for function `delta2bbox` expects type as tensor. Try as following:
```
        # get origin input shape to support onnx dynamic shape
        if torch.onnx.is_in_onnx_export():
            # get shape as tensor
            img_shape = torch._shape_as_tensor(img)[2:]
            img_metas[0]['img_shape_for_onnx'] = img_shape
```

Reply:
> Argument `img_shape` for function `delta2bbox` expects type as tensor. Try as following:
> 
> ```
>         # get origin input shape to support onnx dynamic shape
>         if torch.onnx.is_in_onnx_export():
>             # get shape as tensor
>             img_shape = torch._shape_as_tensor(img)[2:]
>             img_metas[0]['img_shape_for_onnx'] = img_shape
> ```

that's my fault, i'll chek it